test/integration-test.py

```diff
@@ -83,6 +81,80 @@ def expect(target_time, num, table):
     expect(now, 0, "authz")
     expect(after_grace_period, 1, "authz")
 
+def run_janitor():
+    # Set the fake clock to a year in the future such that all of the database
+    # rows created during the integration tests are older than the grace period.
+    now = datetime.datetime.utcnow()
+    target_time = now+datetime.timedelta(days=+365)
+
+    e = os.environ.copy()
+    e.setdefault("GORACE", "halt_on_error=1")
+    e.setdefault("FAKECLOCK", fakeclock(target_time))
+
+    # Note: Must use exec here so that killing this process kills the command.
+    cmdline = "exec ./bin/boulder-janitor --config test/config-next/janitor.json"
+    p = subprocess.Popen(cmdline, shell=True, env=e)
+    p.cmd = cmdline
+
+    # Wait for the janitor to come up
+    waitport(8014, "boulder-janitor", None)
+
+    def statline(statname, table):
+        return "janitor_{0}{{table\"{1}\"}}".format(statname, table)
```

Comment:
I think this winds up looking for, e.g. `janitor_workbatch{table"certificateStatus"}` (i.e. missing an =). Am I missing something?

Also if you drop the final `}` it would be a little more stable in case we add other labels in the future.

Reply:
Good catch, fixed both in 693aacc